indigo_app/views/works.py

```diff
@@ -241,14 +243,21 @@ class WorkOverviewView(WorkViewBase, DetailView):
 
     def get_context_data(self, **kwargs):
         context = super(WorkOverviewView, self).get_context_data(**kwargs)
+        work_tasks = Task.objects.filter(work=self.work)
 
-        context['active_tasks'] = Task.objects\
-            .filter(work=self.work)\
-            .exclude(state='done')\
+        context['active_tasks'] = work_tasks.exclude(state='done')\
             .exclude(state='cancelled')\
             .order_by('-created_at')
         context['work_timeline'] = self.get_work_timeline(self.work)
 
+        completed_tasks = work_tasks.filter(state='done')
+        context['participating_users'] = User.objects\
+            .filter(Q(submitted_tasks__in=completed_tasks) | Q(reviewed_tasks__in=completed_tasks))\
+            .distinct()\
+            .annotate(
+                task_count=Count(Q(submitted_tasks__in=completed_tasks), distinct=True) + Count(Q(reviewed_tasks__in=completed_tasks), distinct=True)
+            ).order_by('-task_count')
+
```

Comment:
@goose-life this query is supposed to get a list of users that have completed and/or review tasks for the specific work and the number of tasks that a user has completed and/or reviewed. Then order the list of users by the number of tasks handled for a particular work.

The problem I'm facing is, if a user has both completed and reviewed a single work, that is counted as 2 works instead of 1.

Reply:
@longhotsummer won't actions help here too?

Reply:
> The problem I'm facing is, if a user has both completed and reviewed a single work, that is counted as 2 works instead of 1.
This rarely happens, don't worry about it too much, it's not a trainsmash.

Yes, I think querying activity objects here may make this easier.